ansible/roles/test/tasks/warm-reboot.yml

```diff
@@ -7,3 +7,5 @@
   include: advanced-reboot.yml
   vars:
       reboot_type: warm-reboot
```

Comment:
The original test case if for happy path. How about still keep the happy one, and add new test cases for various sad paths?

BTW, sad pass -> sad path

Reply:
moved the sad path to a separate yml